src/test/resources/liquibase/harness/snapshot/changelogs/cloudspanner/snapshotCatalogAndSchema.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="1" author="spanner-liquibase-tests">
+        <createTable tableName="snapshot_test_table">
```

Comment:
Is this consistent with the actual test? The name seems to indicate that it tests snapshotting a catalog and schema. The default schema and catalog is the empty string, so it is very possible that this actually covers what it should, so please regard this comment just as a question 'does this really cover what it should, if yes, then just close this comment'.